packages/compiler/test/checker/operations.test.ts

```diff
@@ -34,6 +35,43 @@ describe("compiler: operations", () => {
     strictEqual(b.sourceOperation, a);
   });
 
+  it("operation reference paramters are spread in target operation", async () => {
+    testHost.addTypeSpecFile(
+      "main.tsp",
+      `
+      @test op a(one: string, two: string): void;
+      @test op b is a;
```

Comment:
Should we be testing the same scenario for template instantiation, or is that coming ina  later PR (applies to interfaces and models as well, obviously)

Reply:
i guess you mean `op foo is TemplateOp<T>` , i'll add a test for that.